x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/utils.ts

```diff
@@ -90,7 +92,7 @@ export function crawlConfigServerToClient(crawlConfig: CrawlConfigFromServer): C
   };
 }
 
-export function crawlerEventServerToClient(event: CrawlEventFromServer): CrawlEvent {
+export function crawlEventServerToClient(event: CrawlEventFromServer): CrawlEvent {
```

Comment:
We also have a function called `crawlerEventServerToClient`, should we also rename that?

Reply:
This was already renamed on master as a part of #119541. Thought I reverted that change in 
1b80827 I re-implemented it as a part of ea61a32.  See here: https://github.com/elastic/kibana/pull/119645/commits/ea61a32d350174703b6fd8c88333e465bdb91243#diff-61810b54ca362969682e4268fa842642f3b2e1c611fb62f76e1027b6383a47c1L93

Reply:
Ah sorry. GHs diff view shows it slightly incorrectly for some reason:

<img width="921" alt="Screen Shot 2021-11-24 at 20 24 13" src="https://user-images.githubusercontent.com/809707/143301317-34b1db78-65f8-43fc-9d02-b0cf3615fed3.png">

Reply:
Perhaps because I reverted then re-implemented 😅